OOI_database_seed.json

```diff
@@ -1096,5 +1096,44 @@
       },
       "consult_api": false
     }
+  },
+  {
+    "model": "tools.ooiinformation",
+    "pk": "KATFindingType|KAT-NO-DMARC",
+    "fields": {
+      "last_updated": "2021-12-07T17:24:09.835Z",
+      "data": {
+        "description": "This hostname does not have a DMARC record.",
+        "recommendation": "Set a DMARC record to protect your domain.",
+        "risk": "Medium"
+      },
+      "consult_api": false
+    }
+  },
+  {
+    "model": "tools.ooiinformation",
+    "pk": "KATFindingType|KAT-NO-DKIM",
+    "fields": {
+      "last_updated": "2021-12-07T17:24:09.835Z",
+      "data": {
+        "description": "This hostname does not support a DKIM record.",
```

Comment:
"does not support" vs "does not contain"?

Reply:
I copied this text from internet.nl. The test is not if it contains DKIM like the other checks but if it support them. Whether they really exist can only be tested using the dkimselector object for which @trideeindhoven created a PR